shared/settings/account/add-modals.tsx

```diff
@@ -31,6 +31,13 @@ export const Email = () => {
       dispatch(RouteTreeGen.createClearModals())
     }
   }, [addEmailInProgress, addedEmail, dispatch])
+  // clean on edit
+  React.useEffect(() => {
+    if (email !== addEmailInProgress) {
+      onAddEmailInProgress('')
```

Comment:
I don't understand what you're doing with `onAddEmailInProgress('')`

Reply:
Just resetting it, although it isn't strictly necessary.